benchmark_test.go

```diff
@@ -218,3 +220,106 @@ func TestBenchmarkStatsPerPeer(t *testing.T) {
 	})
 	assert.Equal(t, want, statsServer.Aggregated(), "unexpected stats")
 }
+
+// Method that tests the 'Format' option of Benchmark Options
+func TestBenchmarkOutput(t *testing.T) {
+	type BenchmarkParameters struct {
```

Comment:
you don't need to re-define the structure here, should be able just use the ones you defined in Benchmark.go